src/shared/domain/definition/case-field.model.ts

```diff
@@ -25,4 +25,12 @@ export class CaseField implements Orderable {
 
   @Type(() => WizardPageField)
   wizardProps?: WizardPageField;
+
+  isCollection(): boolean {
+    return this.field_type && this.field_type.type === 'Collection';
+  }
+
+  isComplex(): boolean {
+    return this.field_type && this.field_type.type === 'Complex';
+  }
```

Comment:
@tzarsmango I know is not part of the ticket, but I was thinking whether we could also add other methods like isLabel() (I think to remember that's another one very often used:     return field.field_type.type === 'Label';
) and some others you can think of
Feel free to ignore

Reply:
and a method
Also a method:
getComplexSubfield(path) that allows retrieval of a Complex field subfield by its path.
This logic currently sits in  wizard-page-field-to-case-field.mapper.ts and could be moved in CaseField.

@tzarsmango if you decide to not tackle these as part of this story, please let me know I will create a separate ticket